batch/1/falcon_script_for_5000_data.py

```diff
@@ -0,0 +1,124 @@
+# Description:
```

Comment:
- just noticed, please add a `README.md` to your repository 
- give very minimalist overview of repo for now in `README.md` 
- give brief overview of repo structure
- in this `README.md` : explain what folder and scripts in `batch/` are used